app/javascript/angular/code/services/_mobile_sessions.js

```diff
@@ -245,7 +219,7 @@ export const mobileSessions = (
           cache: true,
           params: { sensor_name: sensorName }
         })
-        .success(callback(session, allSelected));
```

Comment:
this is a callback to calculate bounds which I adjusted to only expect one id now.